www/includes/easyparliament/templates/html/mp/votes_svg.php

```diff
@@ -0,0 +1,45 @@
+<?php
+    $all_lines = array();
+    foreach ($segment['votes']->positions as $key_vote) {
+        if ( $key_vote['has_strong'] || $key_vote['position'] == 'has never voted on' ) {
+            $text = ucfirst(strip_tags($key_vote['desc']));
+        } else {
+            $text = "We don&rsquo;t have enough information to calculate $full_name &rsquo;s position on " . $key_vote['policy'] . ".";
```

Comment:
Since people are using this image to share what their MP voted on, I wonder whether we should just silently ignore policies on which we don't hold enough data? I can’t see much value in us announcing that “we don’t have enough information” here – better just to ignore that policy, and stick to showing the ones that we *do* have the data for, because those will be the ones the person sharing the image really cares about.